src/codewhisperer/util/dependencyGraph/rubyDependencyGraph.ts

```diff
@@ -0,0 +1,226 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { existsSync, statSync, readdirSync } from 'fs'
```

Comment:
Try to use `FileSystemCommon.instance` instead of `fs` since we will want to reduce the amount of work when making CW compatible in the browser (`fs` is not browser compatible)